cmd/activator/main.go

```diff
@@ -213,6 +216,14 @@ func main() {
 	}
 	ah = reqLogHandler
 
+	ah = handler.TimeToFirstByteTimeoutHandler(ah, "activator request timeout", func(r *http.Request) time.Duration {
+		rev := util.RevisionFrom(r.Context())
+		if rev != nil && rev.Spec.TimeoutSeconds != nil {
```

Comment:
```suggestion
		if rev := util.RevisionFrom(r.Context()); rev != nil && rev.Spec.TimeoutSeconds != nil {
```
Still wondering why did you pick this exact place to put the handler? E.g. not after reqLogHandler or trace handler?

Reply:
No particular reason, we can move it further down.